src/plone/restapi/services/linkintegrity/configure.zcml

```diff
@@ -0,0 +1,17 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:plone="http://namespaces.plone.org/plone"
+    xmlns:zcml="http://namespaces.zope.org/zcml"
+    >
+
+  <include package="Products.CMFEditions" />
+
+  <plone:service
+      method="POST"
```

Comment:
@ericof @jensens it makes sense to use POST instead of GET here because we do not want to cache that endpoint, right? I guess the URL parameter size restriction is not a matter in real life here.

Reply:
Second though: the linkintegrity endpoint does not change anything, so maybe GET would be the better fit after all. What do you think?

Reply:
so, if it does not change anything and it does not need complex datastructures as parameters a POST is pointless.

Reply:
Ok, moved to get. We need to update the frontend pr too..i'm going to notify the change there